runtime/runc_jailer.go

```diff
@@ -294,13 +294,13 @@ func (j *runcJailer) BuildLinkFifoHandler() firecracker.Handler {
 	return firecracker.Handler{
 		Name: jailerFifoHandlerName,
 		Fn: func(ctx context.Context, m *firecracker.Machine) error {
-			logFifo, err := j.makeLinkInJail(m.Cfg.LogPath)
+			logFifo, err := j.makeLinkInJail(m.Cfg.LogPath, internal.FirecrackerLogFifoName)
```

Comment:
I want to phrase this change to ensure I'm understanding it correctly:

 What's happening here is that previously we were accepting a LogPath, and using the customer provided filename and now we will be using our own filename to avoid customers accidentally using a name that conflicts with other important names? 

Under what circumstances would this change be visible to current users?

Reply:
I think the change won't be visible to users, except for the one who are affected by the issue and workaround that by providing "safe" paths that don't conflict.